src/components/LocalityListItem/LocalityListItem.js

```diff
@@ -17,13 +17,18 @@ const LocalityListItem = ({ locality, onClick, onMouseEnter, onMouseLeave }) =>
       onClick={handleClick}
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
-      className={`list-item dmg-${dmgGrade}`}
+      className={`${Styles.listItem} Styles.dmg-${dmgGrade}`}
       style={{}}
     >
-      <div className="list-item-header">{locName}, {stateName}</div>
-      <div className="list-item-metrics">
-        <div><span className="label">MARG. SOCIAL</span> <span className="value">{margGrade || '?'}</span></div>
-        <div><span className="label">VIVIENDAS DAÑADAS</span> <span className="value">{fmtNum(total)}</span></div>
+      <div className={Styles.listItemWrapper}>
+        <div className={`${Styles.listItemHeader}`}>
+          <div className={Styles.locName}>{locName}, </div>
```

Comment:
Try putting a `span` inside a `div`, and styling the span with `Styles.stateName` differently. Otherwise the space after `locName` will get eaten